addon/components/frost-object-browser-filter.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember'
+import layout from '../templates/components/frost-object-browser-filter'
+import _ from 'lodash'
+
+function callIfDefined (context, functionName, ...args) {
```

Comment:
Yeah, it would be handy. If not in ember core, we could make a utility library that handles a bunch of the stuff we keep repeating.